staff/dtiburcio/leadtech-site-semantic-exercise/index.html

```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+
+</head>
+<body>
+<header>
+    <h1><a href="https://www.leadtech.com">Leadtech</a></h1>
+    <h2>MENU</h2>
+    <nav>
+        <ul>
+            <li><a href="http://www.leadtech.com">Leadtech</a></li>
+            <li><a href="http://www.leadtech.com/about-us">about us</a></li>
+            <li><a href="https://leadtech.com/work-with-us">work with us</a>
+                <ul>
+                    <li><a href="https://leadtech.com/work-with-us/backend">Backend</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/bi">BI</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/content-and-copywriting">Content and copywriting</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/customer-service">Customer Service</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/finance">Finance</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/frontend">Frontend</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/human-resources">Human Resources</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/project-manager">Project Management</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/qa">QA</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/sem">SEM</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/seo">SEO</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/social-media">Social Media</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/sysadmin">Sysadmin</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/ux">UX</a></li>
+                </ul>
+            </li>
+            <li><a href="https://leadtech.com/entrepreneur">entrepreneurs</a></li>
+            <li><a href="https://leadtech.com/investor">investors</a></li>
+            <li><a href="https://leadtech.com/contact">contact</a></li>
+        </ul>
+    </nav>
+</header>
+<body>
+<section>
+    <h1>hi, good night from leadtech</h1>
+    <h2>The company that knows how to take an idea and turn it into a success in style.</h2>
+    <aside>
+        <p>
```

Comment:
The tag `blockquote` can be used to represent quotes → https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote